### DISCUSSION TITLE
List of Plugins

### DISCUSSION THREAD

**Discussion by zzenonn** [Impostor/Impostor#291] [Ideas]
I think it would be helpful if there is a maintained list of existing plugins written by the community? Some plugins I found on GitHub:

https://github.com/VisualEnterprise/AmongUsDisguise
https://github.com/oliver4888/Impostor-LobbyCommands
https://github.com/dimaguy/ImpostorHQ

**Comment by omalleyian** [comment]
You can check the dependents page for a basic list: https://github.com/Impostor/Impostor/network/dependents.
